blackbox/docs/sql/system.txt

```diff
@@ -1092,6 +1092,10 @@ where ``N`` is the number of nodes in the cluster, and ``M`` is the value of
 the setting :ref:`discovery.zen.minimum_master_nodes
 <discovery.zen.minimum_master_nodes>`.
 
+You can change the value by issuing the following SQL statement::
+
+  SET GLOBAL discovery.zen.minimum_master_nodes = M;
```

Comment:
synopsis is: `SET GLOBAL [ PERSISTENT | TRANSIENT ] setting_ident = value [ , ... ]`